### PR TITLE
Update organisation types

### PR DESCRIPTION
**Pull request by yeahfro**
## Description of change
Organisation types were previously hardcoded and need to be set from a subset of business types.

## Test instructions
1. Browse to `~/companies/create`
2. Select `UK`
3. Click `Next`
4. Search for `ralph`
5. Click `I cannot find the company I am looking for`
6. Click `I still cannot find the company`, you can see the `Organisation type` radio buttons

### Before
Screenshot 2019-09-13 at 15 05 15

### After 
Screenshot 2019-09-13 at 15 04 27

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to develop?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/companies/apps/add-company/repos.js [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
+  const businessTypeOptions = await getOptions(token, 'business-type')
+  const filteredBusinessTypeOptions = filter(businessTypeOptions, ({ value }) => {
+    return includes(BUSINESS_TYPE_WHITELIST, value)
+  })
+
+  return sortBy(map(filteredBusinessTypeOptions, (businessTypeOption) => {
+    if (businessTypeOption.label === COMPANY_LABEL) {
+      return {
+        ...businessTypeOption,
+        label: LIMITED_COMPANY_LABEL,
```
Comment (reupen) [inline src/apps/companies/apps/add-company/repos.js]: Is this temporary or are we going to rename it in the database later?

Comment (yeahfro) [reply src/apps/companies/apps/add-company/repos.js]: Is it safe to rename without affecting other dependents?

File: src/apps/companies/apps/add-company/repos.js [hunk 2]
```diff
@@ -1,14 +1,43 @@
-const { filter } = require('lodash')
+const { filter, map, includes, sortBy } = require('lodash')
 
 const { getOptions } = require('../../../../lib/options')
 
 const METADATA_COUNTRY_UK_LABEL = 'United Kingdom'
+const COMPANY_LABEL = 'Company'
+const LIMITED_COMPANY_LABEL = 'Limited company'
+const BUSINESS_TYPE_WHITELIST = {
+  charity: '9dd14e94-5d95-e211-a939-e4115bead28a',
```
Comment (benpbenp) [inline src/apps/companies/apps/add-company/repos.js]: Are these hardcoded database ids? Why is it necessary, can't we just send the name and have the backend figure out the database id? (This comment applies to all the other points were these ids appear too)

Comment (yeahfro) [reply src/apps/companies/apps/add-company/repos.js]: The name can change but the ID will not.

File: src/apps/companies/apps/add-company/repos.js (hunk 1 above)
Comment (reupen) [reply src/apps/companies/apps/add-company/repos.js]: Not sure if anything is currently depending on the name (the old 'add company' flow might be I guess).

**Pull request merged**